beacon_node/src/config.rs

```diff
@@ -382,10 +384,30 @@ pub fn get_config<E: EthSpec>(
         client_config.execution_layer = Some(el_config);
     }
 
+    // 4844 params
+    client_config.trusted_setup = context
+        .eth2_network_config
+        .as_ref()
+        .and_then(|config| config.kzg_trusted_setup.clone());
+
+    // Override default trusted setup file if required
+    // TODO: consider removing this when we get closer to launch
+    if let Some(trusted_setup_file_path) = cli_args.value_of("trusted-setup-file-override") {
```

Comment:
Is it still worth keeping this flag?

Reply:
I believe this is still being used by EF devops so I think we need it for now